src/screens/ChatScreen/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from 'react';
+import { StyleSheet, KeyboardAvoidingView, SafeAreaView, Platform } from 'react-native';
+import { useDispatch } from 'react-redux';
+
+import { ChatList } from 'app/screens/ChatScreen/components/ChatList';
+import { InputMessage } from 'app/screens/ChatScreen/components/InputMessage';
+import { getFirestoreCollection } from 'app/store/thunk';
+import * as Types from 'app/types';
+
+export type Props = Types.RootStackScreenProps<Types.Route.Chat>;
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+  },
+});
+
+const OS = Platform.OS;
+const keyboardBehavior = OS === 'ios' ? 'padding' : 'height';
+const keyboardVerticalOffset = OS === 'ios' ? 100 : 80;
```

Comment:
How it would work on old Iphones, without notch? 
if it's related to HomeIndicator at the bottom, you would need some check if the device has it.
Test it on iPhone SE so you will know what I mean :)